src/domain/templates/components/TemplateSelectors/SubspaceTemplateSelector.tsx

```
@@ -0,0 +1,79 @@
+import Gutters, { GuttersProps } from '@/core/ui/grid/Gutters';
+import { useSpace } from '@/domain/journey/space/SpaceContext/useSpace';
+
+interface SubspaceTemplateSelectorProps extends GuttersProps {
+  name: string;
+}
+
+export const SubspaceTemplateSelector: FC<SubspaceTemplateSelectorProps> = ({ name, ...rest }) => {
+  const { t } = useTranslation();
+  const { spaceId, loading: loadingSpace } = useSpace();
+  const [isDialogOpen, setDialogOpen] = useState(false);
+  const [field, , helpers] = useField<String>(name);
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)